dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/step/DescribeStep.java

```diff
@@ -110,13 +111,19 @@ private void readField(InProgressSubmission obj, SubmissionStepConfig config, Da
     }
 
     @Override
-    public void doPatchProcessing(Context context, Request currentRequest, InProgressSubmission source, Operation op)
-        throws Exception {
+    public void doPatchProcessing(Context context, Request currentRequest, InProgressSubmission source, Operation op,
+                                  SubmissionStepConfig stepConf) throws Exception {
 
         PatchOperation<MetadataValueRest> patchOperation = new PatchOperationFactory()
-            .instanceOf(DESCRIBE_STEP_METADATA_OPERATION_ENTRY, op.getOp());
-        patchOperation.perform(context, currentRequest, source, op);
-
+                        .instanceOf(DESCRIBE_STEP_METADATA_OPERATION_ENTRY, op.getOp());
+        DCInputSet inputConfig = inputReader.getInputsByFormName(stepConf.getId());
+        String[] split = patchOperation.getAbsolutePath(op.getPath()).split("/");
+        if (inputConfig.isFieldPresent(split[0])) {
+            patchOperation.perform(context, currentRequest, source, op);
+        } else {
+            throw new UnprocessableEntityException("The attribute " + split[0] + " does not present in section "
```

Comment:
I think this error should say:  `"The field " + split[0] + " is not present in section "`